Restore compatibility with Brahma v1 payloads

d0k3 opened the pull request:
Following up yesterdays discussion, I reverted the change in arm11.s, so that again the ARM9 entry point is stored as intended. Reverting this doesn't have any averse effects on boot success rates (tested with Decrypt9 & ReiNAND CFW). Also, @patois sample code reboots as intended again.

d0k3 committed:
Restore compatibility with Brahma v1 payloads

Thanks @patois for pointing this out!

jasondellaluce commented on source/arm11.s:
```diff
@@ -88,7 +94,7 @@ wait_arm11_loop:
 	BX              R1
 
 	pa_hijack_arm9_dst:  .long 0x1FFFFC00
-	pa_arm11_code:       .long 0x1FFFFFF8
```
Here is what i was talking about.
This address has nothing to do with the brahma 1 compatibility, since this is the pointer where the arm11 entrypoint will be written to by arm9.
Keeping this to 0x1FFFFFFC (as original) makes the arm11 kernel load the original firm partition after the firmlaunch, which causes any cfw loader to run an updated arm9 kernel with an outdated arm11 one in the otherside.
So, leaving it 0x1FFFFFF8 should not affect in any way Brahma1 compatibility, while improves stability if you wish to run custom code on the arm11.
Can youy please change this in another pull request?

d0k3 commented:
Okay, I got it, thanks for the explanation! I fixed it via an additional commit and just squashed that commit into the old one - I hope that is okay as well. This commit also contains two minor cosmetic fixes, hope thats alright as well.

delebile commented:
In theory, the only real change that should be done to patois example in order to work is the following, which can be written in C in the main function, or in asm in start.s before of after being jumped to main.

> [C]
*(unsigned int)0x1ffffff8 = *(unsigned int)0x1ffffffC;

> [ASM]
ldr r0, =0x1ffffffC
ldr r0, [r0]
ldr r1, =0x1ffffff8
str r0, [r1]

Try this out.

d0k3 commented:
Okay, I tried it and it works. I'm pretty sure you are not surprised :). This change doesn't break old Brahma 1 compatibility, correct? Because this only makes a copy of the data to another place.

You can close this pull request, don't take it over cause it will not help and might even lower boot rates (I am not sure). I'll make a second pull request with updated sample projects, though.

delebile commented:
That's good then :)
